okio/jvm/src/test/java/okio/BufferedSourceTest.java

```diff
@@ -882,7 +882,7 @@ private void assertLongDecimalString(String s, long expected) throws IOException
   }
 
   @Test public void select() throws IOException {
-    Options options = Options.of(
+    Options options = Options.Companion.of(
```

Comment:
`Companion` is redundant (this was probably auto-generated, but still)

Reply:
Will fix in follow-up.